processor/core/src/main/java/com/redhat/service/smartevents/processor/sources/aws/AwsS3Source.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.service.smartevents.processor.sources.aws;
+
+import com.redhat.service.smartevents.processor.GatewayBean;
+
+public interface AwsS3Source extends GatewayBean {
+
+    String TYPE = "AwsS3";
+
+    String BUCKET_NAME_OR_ARN_PARAMETER = "bucket_name_or_arn";
+    String REGION_PARAMETER = "region";
+    String ACCESS_KEY_PARAMETER = "access_key";
+    String SECRET_KEY_PARAMETER = "secret_key";
```

Comment:
I would name these properties as `aws_region`, `aws_access_key_id` and `aws_secret_access_key` only because the official `aws` CLI uses this naming so I assume these are familiar among AWS users